tools/CMakeLists.txt

```diff
@@ -1,3 +1,7 @@
+include(LLVMTools)
+llvm_find_config(3.4)
+if (LLVM_CONFIG_EXECUTABLE)
+    add_subdirectory(typelib-clang-tlb-importer)
```

Comment:
maybe having a message here when no llvm is found and thus the importer is not built?